src/App.js

```diff
@@ -1,10 +1,47 @@
 import React from 'react';
 import './App.css';
+import handy from "./assets/bag_1.png";
+import stylish from "./assets/bag_2.png";
+import simple from "./assets/bag_3.png";
+import trendy from "./assets/bag_4.png";
+import brand from "./assets/brand.png";
+import story from "./assets/our_story.png";
+import Button from "./components/Buttons";
+import Products from "./components/Products";
+import Tiles from "./components/Tiles";
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  );
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button buttonText="to the collection"/>
+                <Button buttonText="shop all bags"/>
+                <Button buttonText="pre-order"/>
+            </nav>
+            <main>
+                <Products advice="Best seller" productImage=<img src={handy} alt="Handy bag"/> articleName="The handy
+                bag" price="€450,-"/>
```

Comment:
Voor de prijs zou ik jezelf altijd tenminste het euro teken typen besparen door dat al in het component te zetten. Heeft natuurlijk wel als effect dat naar dollars switchen lastiger is.

prijs = `€${price}`